packages/web-api/src/controllers/scan-request-controller.ts

```diff
@@ -19,7 +20,7 @@ export class ScanRequestController extends ApiController {
         @inject(webApiIocTypes.azureFunctionContext) protected readonly context: Context,
         @inject(ScanDataProvider) private readonly scanDataProvider: ScanDataProvider,
         @inject(GuidGenerator) private readonly guidGenerator: GuidGenerator,
-        @inject(ServiceConfiguration) private readonly serviceConfig: ServiceConfiguration,
+        @inject(ServiceConfiguration) protected readonly serviceConfig: ServiceConfiguration,
```

Comment:
why did you make this protected?

Reply:
This is protected in super class. Same as logger.